diff --git a/Foo.php b/Foo.php
index 114dd85..e05f61b 100644
--- a/Foo.php
+++ b/Foo.php
@@ -1,5 +1,7 @@
 <?php
 
+namespace Acme\Tools;
+
 class Foo
 {
     public function doAwesomeThings()
diff --git a/some-other-file.php b/some-other-file.php
index e094b7f..43c1ab7 100644
--- a/some-other-file.php
+++ b/some-other-file.php
@@ -2,6 +2,6 @@
 
 require 'Foo.php';
 
-$foo = new Foo();
+$foo = new \Acme\Tools\Foo();
 
 $foo->doAwesomeThings();
