diff --git a/Foo.php b/Foo.php
index e05f61b..c5fbc7a 100644
--- a/Foo.php
+++ b/Foo.php
@@ -7,5 +7,8 @@ class Foo
     public function doAwesomeThings()
     {
         echo "Hi Foo!\n";
+
+        $dt = new DateTime();
+        echo $dt->getTimestamp()."\n";
     }
 }
diff --git a/some-other-file.php b/some-other-file.php
index ad92b02..830e25b 100644
--- a/some-other-file.php
+++ b/some-other-file.php
@@ -7,6 +7,3 @@ use Acme\Tools\Foo;
 $foo = new Foo();
 
 $foo->doAwesomeThings();
-
-$dt = new DateTime();
-echo $dt->getTimestamp()."\n";
