diff --git a/Foo.php b/Foo.php
index 5cc1c66..55fc585 100644
--- a/Foo.php
+++ b/Foo.php
@@ -2,7 +2,7 @@
 
 namespace Acme\Tools;
 
-use \DateTime;
+use DateTime;
 
 class Foo
 {
diff --git a/some-other-file.php b/some-other-file.php
index 718bc7a..9b3d59b 100644
--- a/some-other-file.php
+++ b/some-other-file.php
@@ -2,7 +2,7 @@
 
 require 'Foo.php';
 
-use \Acme\Tools\Foo;
+use Acme\Tools\Foo;
 
 $foo = new Foo();
 
